modules/Hoot/ui/modals/importDataset.js

```diff
@@ -363,6 +363,16 @@ export default class ImportDataset {
                 Hoot.events.emit( 'modal-closed' );
             } );
     }
+    checkLayerName(layerName) {
+        var matches = [];
+        for (let i = 0; i < Hoot.layers.allLayers.length; i++) {
+            var checkedLayer = Hoot.layers.allLayers[i].name;
+            if (checkedLayer === layerName || new RegExp(layerName + '_\\d+').test(checkedLayer) ) {
```

Comment:
The `_\\d+` regex here also matches with the generated UUID suffixes used by conflation.  Could we make it `OutputName (2)` like browser downloads do when a filename already exists?